### ISSUE TITLE
save model

### ISSUE THREAD

**Issue by fujiaixin09** [JisongXie/StegaStamp_pytorch#1]
Hello! Is your code incomplete? I didn't find save_ model in your code?

**Comment by JisongXie** [comment]
Hi，my code needs to be improved. But, you mean my trained model? I don't put it here.

**Comment by fujiaixin09** [comment]
I mean, your code doesn't save the model after running, and there are no statements to save in your code. Therefore, is your code incomplete?

**Comment by JisongXie** [comment]
Yes, it's incomplete. There are no statements to save model in my code. I am busy now, so there is almost no time to update it.

**Comment by fujiaixin09** [comment]
That's OK. Thank you very much!
I tried to reproduce the "stegastamp" project and train my own model, but I found that my accuracy was less than 0.95, but my super parameters and code were provided by the author, and the configuration of the environment was almost the same as that of the author. Do you know why?

**Comment by JisongXie** [comment]
is your dataset the same as the author's? is your training accuracy less than 0.95, or your test accuracy?

**Comment by fujiaixin09** [comment]
The data set I used was 10W pictures.，like the data set used by the author, my training accuracy was less than 0.95

**Comment by JisongXie** [comment]
you can use the dataset mentioned in the paper to try, I remember that training accuracy can go up to nearly 0.97.

**Comment by fujiaixin09** [comment]
Yes, I use the data set published by the author, but the training accuracy is less than 0.95.Is there any skill in training this network? I put all the super parameters into the training directly.

**Comment by JisongXie** [comment]
I use the author's tensorflow code to train, it can reach up to 0.97. My pytorch code haven't finished,  so it need to be improved. e.g., there is no gradient clip in my code, there is no model saving in my code, etc.

**Comment by fujiaixin09** [comment]
Well, thank you very much. But the training accuracy of my reproduction is less than 0.95, and now I can't find the reason.

**Comment by JisongXie** [comment]
you use your reproduced code to run it? pytorch or tensorflow?

**Comment by fujiaixin09** [comment]
The author published the code, tensorflow.I don't know what's wrong

**Comment by JisongXie** [comment]
is it far below 0.95? if it's near 0.95, maybe it's no problem. maybe you can adjust your learning rate or use learning rate scheduler.

**Comment by fujiaixin09** [comment]
My final training accuracy is 0.91, so how can I change the learning rate?

**Comment by JisongXie** [comment]
of course, try samller learning rate after you reach the accuracy 0.91, to see whether it will go up.

**Comment by fujiaixin09** [comment]
OK, I'll try it. Thank you
